{"name": "test_normal_tx_to_script_addr", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/home/martink/Source/repos/cardano-node-tests2/tmp/testrun_20201102.log'", "trace": "tmp_path_factory = TempPathFactory(_given_basetemp=PosixPath('/tmp/pytest-of-martink/pytest-2/popen-gw2'), _trace=<pluggy._tracing.TagTracerSub object at 0x7fce2fdc2d90>, _basetemp=PosixPath('/tmp/pytest-of-martink/pytest-2/popen-gw2')), worker_id = 'gw2'\nrequest = <SubRequest 'cluster_manager' for <Function test_normal_tx_to_script_addr>>\n\n    @pytest.yield_fixture\n    def cluster_manager(\n        tmp_path_factory: TempdirFactory,\n        worker_id: str,\n        request: FixtureRequest,\n    ) -> Generator[parallel_run.ClusterManager, None, None]:\n>       cluster_manager_obj = parallel_run.ClusterManager(\n            tmp_path_factory=tmp_path_factory, worker_id=worker_id, request=request\n        )\n\n/home/martink/Source/repos/cardano-node-tests2/cardano_node_tests/tests/conftest.py:121: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/home/martink/Source/repos/cardano-node-tests2/cardano_node_tests/utils/parallel_run.py:67: in __init__\n    self.lock_log = self._init_log()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <cardano_node_tests.utils.parallel_run.ClusterManager object at 0x7fce2f88ffa0>\n\n    def _init_log(self) -> Path:\n        \"\"\"Return path to run log file.\"\"\"\n        env_log = os.environ.get(\"SCHEDULING_LOG\")\n        if env_log:\n            run_log = Path(env_log).expanduser()\n            if not run_log.is_absolute():\n                # the path is relative to LAUNCH_PATH (current path can differ)\n                run_log = helpers.LAUNCH_PATH / run_log\n            # create the log file if it doesn't exist\n>           open(run_log, \"a\").close()\nE           FileNotFoundError: [Errno 2] No such file or directory: '/home/martink/Source/repos/cardano-node-tests2/tmp/testrun_20201102.log'\n\n/home/martink/Source/repos/cardano-node-tests2/cardano_node_tests/utils/parallel_run.py:82: FileNotFoundError"}, "description": "Send funds to script address using TX signed with skeys (not using witness files).", "start": 1604310980676, "stop": 1604310980676, "uuid": "49c4943e-c023-42e7-bdf6-f3222a17f9d0", "historyId": "d5ad08d55894cc781f5454a2dd37521b", "testCaseId": "ed3ee9fbd17ab2a00f51d97f736d8173", "fullName": "cardano_node_tests.tests.test_multisig.TestBasic#test_normal_tx_to_script_addr", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('temp_dir')"}, {"name": "parentSuite", "value": "cardano_node_tests.tests"}, {"name": "suite", "value": "test_multisig"}, {"name": "subSuite", "value": "TestBasic"}, {"name": "host", "value": "bender-3900x"}, {"name": "thread", "value": "29221-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cardano_node_tests.tests.test_multisig"}], "links": [{"type": "link", "url": "https://github.com/input-output-hk/cardano-node-tests/blob/d28a503424b5f9f0359a9e79ea272dfac0f801bd/cardano_node_tests/tests/test_multisig.py#L314", "name": "https://github.com/input-output-hk/cardano-node-tests/blob/d28a503424b5f9f0359a9e79ea272dfac0f801bd/cardano_node_tests/tests/test_multisig.py#L314"}]}